src/services/tourettes.js

```diff
@@ -16,17 +16,23 @@ const profanities = [
   'feministdemonstration',
   'mr potatomun',
   'fistelfisting',
-  'piss! piss out my ass!!'
+  'piss! piss out my ass!!',
```

Comment:
so did I before I read this: https://medium.com/@nikgraf/why-you-should-enforce-dangling-commas-for-multiline-statements-d034c98e36f8#.3bf9gst57